metricbeat/Makefile

```diff
@@ -6,7 +6,7 @@ TEST_ENVIRONMENT?=true
 ES_BEATS?=..
 
 # Metricbeat can only be cross-compiled on platforms not requiring CGO.
-GOX_OS=netbsd linux windows
+GOX_OS=linux windows
```

Comment:
Umm, why darwin is not here?

Reply:
I think it was not present because it requires CGO and a compiler for darwin.